frontend/components/product-list/sections/FilterableProductsSection/useFilteredRefinementList.tsx

```diff
@@ -10,9 +10,17 @@ export function useFilteredRefinementList(props: UseRefinementListProps) {
    const hitsCount = results?.nbHits ?? 0;
    const isAnyRefined = items.some((item) => item.isRefined);
 
-   const filteredItems = isAnyRefined
+   let filteredItems = isAnyRefined
       ? items
       : items.filter((item) => hitsCount !== item.count);
 
-   return { ...rest, items: filteredItems };
+   let emptyFacetState = false;
+
+   if (filteredItems.length == 0) {
+      filteredItems = items.filter((item) => hitsCount === item.count);
+      if (filteredItems.length > 0) {
+         emptyFacetState = true;
+      }
+   }
```

Comment:
I read this as "If there are no filtered items, but some items were filtered out, then we should gray out this facet", though I'm not sure if I'm correct.

Would this be equivalent?

```suggestion
   const isEmptyFacetState = filteredItems.length === 0 && items.length > 0;
```

Reply:
Resolved d58eee9cc93e7d1bb0ad724515e9eb8c8db7de24